benchmarks/Microsoft.AspNetCore.Routing.Performance/Matchers/MatcherAzureBenchmark.cs

```diff
@@ -1,20 +1,18 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
 using System.Threading.Tasks;
 using BenchmarkDotNet.Attributes;
 
 namespace Microsoft.AspNetCore.Routing.Matchers
 {
     // Generated from https://github.com/Azure/azure-rest-api-specs
-    public partial class AzureMatcherBenchmark : MatcherBenchmarkBase
+    public partial class MatcherAzureBenchmark : MatcherBenchmarkBase
```

Comment:
Renaming and changing these so they can sort alphabetically. It makes the console UI for this much better to use.